modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustServerCodegen.java

```diff
@@ -1056,6 +1056,26 @@ public boolean isDataTypeFile(final String dataType) {
         return dataType != null && dataType.equals(typeMapping.get("File").toString());
     }
 
+    /**
+     * Add operation to group
+     *
+     * @param tag          name of the tag
+     * @param resourcePath path of the resource
+     * @param operation    OAS Operation object
+     * @param co           Codegen Operation object
+     * @param operations   map of Codegen operations
+     */
+    @SuppressWarnings("static-method")
```

Comment:
Can you add an `@Override` annotation here? That way if the method we are overloading changes, we'll get a compile error.

Reply:
ok. I will add it.